### ISSUE TITLE
Bootstrap ACL System From Web UI

### ISSUE THREAD

**Issue by picatz** [hashicorp/nomad#10232]
### Proposal

After doing the manual ACL bootstrap process enough times, I think it'd be nice to be able to perform that action from the web UI as well. A rough idea of what I'm thinking that could look like:

<img width="1920" alt="Screen Shot 2021-03-25 at 10 36 05 AM" src="https://user-images.githubusercontent.com/14850816/112496068-d5e41600-8d5a-11eb-9b10-e2e0bde88d47.png">

### Use-cases

When a Nomad cluster is first provisioned with ACLs configured, a manual (or otherwise automated) ACL bootstrapping step is required. I often open up the UI first before running the bootstrap command, and would love to _not need_ to context switch back to my terminal right away. I imagine being able to perform this action from the UI will probably be most useful in getting started or demo environments, but would be nice to have.

**Comment by DingoEatingFuzz** [comment]
Neat! This does sound nice for dev/demo environments. It would also give us an opportunity to mention best practices and link to Learn.

First question though, is there a way to detect if ACLs aren't bootstrapped yet?

There are three states we'd need to be able to differentiate here:

1. No ACL support
2. ACL support, not bootstrapped
3. ACL bootstrapped, no token set
4. ACL bootstrapped, token set

Right now we determine 1 by parsing the error message when attempting to fetch the `self` token. 4 and 3 we are easy, since a token either is or isn't set. Disambiguating 2 and 3 though I don't think is currently possible.

**Comment by picatz** [comment]
You're right, I don't think there's a way to disambiguating 2 and 3. There probably needs to be a new API endpoint (with accompanying RPC) to check if the cluster is bootstrapped? 🤔 

 Something like:
 ```console
 $ curl -X POST localhost:4646/v1/acl/bootstrap/check
 {
     "bootstrapped":false, 
     "bootstrappable": true
}
 ```
 

That would also be useful for other tools that might want to check that state of the cluster like an [InSpec resource](https://github.com/picatz/inspec-nomad).

```ruby
describe nomad do
    it { should be_bootstrapped }
end
```
